routingpy/routers/opentripplanner_v2.py

```diff
@@ -210,53 +178,47 @@ def _parse_legs(self, legs):
     def isochrones(
         self,
         locations: List[float],
-        profile: Optional[str] = "WALK,TRANSIT",
-        time: Optional[datetime.datetime] = datetime.datetime.now(datetime.timezone.utc),
-        cutoffs: Optional[List[int]] = [3600],
+        profile: str,
+        intervals: List[int],
+        interval_type: Optional[str] = None,
```

Comment:
same here: I changed those to be consistent with other routers

Reply:
Yes, I was planning to do that in the 2nd phase. I wanted the 1st version to reflect the router's API as closely as possible.

Reply:
Ah ok, that'd have been fine too, as long as we don't release, which we have already. It's not that bad though, the next release will break the date_time stuff anyways, so we'll need to increment major version.